js/samples/06.auth.adaptiveCardSSO/src/graphClient.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Client } from '@microsoft/microsoft-graph-client';
+import 'isomorphic-fetch';
+
+/**
+ * This class is a wrapper for the Microsoft Graph API.
+ * See: https://developer.microsoft.com/en-us/graph for more information.
+ */
+export class GraphClient {
+    graphClient: Client;
+    _token: string;
+
+    constructor(token: string) {
+        if (!token || !token.trim()) {
+            throw new Error('GraphClient: Invalid token received.');
+        }
+
+        this._token = token;
+
+        // Get an authenticated Microsoft Graph client using the token issued to the user.
+        this.graphClient = Client.init({
+            authProvider: (done) => {
+                done(null, this._token); // First parameter takes an error if you can't get an access token.
+            }
+        });
+    }
+
+    public async GetMyProfile() {
+        return await this.graphClient.api('/me').get();
+    }
+
+    // Gets the user's photo
+    public async GetPhotoAsync(): Promise<string> {
```

Comment:
Should this be `getProfilePhotoAsync` per MS graph API?

Reply:
fixed